src/main/webapp/user_profile.html

```diff
@@ -115,5 +49,25 @@ <h1>CREATE A NEW TASK: </h1>
             </form>
         </div>
     </div>
+    <div id="editTaskModal">
+        <div id="editModal">
```

Comment:
editModal smells more like a class than an ID; will there be other entities that users can edit within a modal?

Reply:
Emm I don't think the users edit other entities within the modal. The modal should only serve to edit tasks. Thus, I rename the ids of the two `<div>` element in the new commit. The outer `<div>` is now named as `editTaskModalWrapper` and the inner `<div>` element is now named as `editTaskModal`.

Reply:
I don't mean that users will edit another kind of entity in *this* modal, just that you might have other entities in the future that you want to let users edit in a modal. At that time, it would be handy if this were a class.

I see that you've made classes now, so thanks! (`modalWrapper` and `modal`)